src/kubernetes/skipper/skipper-deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: skipper
+  labels:
+    app: skipper
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: skipper
+    spec:
+      containers:
+      - name: skipper
+        image: springcloud/spring-cloud-skipper-server:1.0.0.BUILD-SNAPSHOT
```

Comment:
Maybe we want to use `1.0.0.M2` here after the Skipper M2 release?

Reply:
@sabbyanandan I think this should be `1.0.0.M2` here.

Reply:
will fix while merging.

Reply:
We might as well change it back to `1.0.0.BUILD-SNAPSHOT` post- SCDF kubernetes release.